lib/interactive.dart

```diff
@@ -194,9 +198,53 @@ class Interactive {
       peopleByLabel[key].add(people);
     });
 
+    for (var i = 0; i < xAxisCategories.length; ++i) {
+      var colSVG = svg.SvgElement.tag("g");
+
+      var colData = peopleByLabel[xAxisCategories[i].value];
+      num colOffset = (i + 0.5) * (chartWidth / xAxisCategories.length);
+
+      for (var j = 0; j < colData.length; ++j) {
+        num x = (j % SQ_IN_ROW - SQ_IN_ROW / 2) * SQ_SIZE + colOffset;
+        num y = (CHART_HEIGHT - CHART_XAXIS_HEIGHT - 1.5 * SQ_SIZE) -
```

Comment:
truncate does the same as floor?

Reply:
Seems to do the same i.e. drops the decimal digits, but i think floor is a lot more explicit. Fixed